.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,5 +5,6 @@
 - [ ] made PR ready for code review
 - [ ] added some tests for the functionality
 - [ ] updated the documentation
+    - [ ] included any new terms in the [glossary](https://github.com/RasaHQ/rasa/blob/master/docs/glossary.rst)
```

Comment:
I would not indent that actually, what do you think?
```suggestion
- [ ] included any new terms in the [glossary](https://github.com/RasaHQ/rasa/blob/master/docs/glossary.rst)
```

Reply:
I highly doubt if any feature that doesn't require a change in the docs would require a glossary item to be changed, so think it'd make sense for it to be hierarchical.

Reply:
I would maybe even add a note to the effect that adding any new terms should be avoided.

Reply:
> I highly doubt if any feature that doesn't require a change in the docs would require a glossary item to be changed, 

Well, then you can also just tick it, right? Having a single item with indent seems weird to me.

> I would maybe even add a note to the effect that adding any new terms should be avoided.

Not sure whether we need this. I think the reviewer should check if any change in the glossary makes sense and intervene if needed.

Reply:
right - but if we include that in the checklist it seems like you should be adding new terms all the time

Reply:
@amn41 Surely the way we are going about it currently is to hope that people exercise common sense, right? Otherwise, even the `updated the documentation"` would be plagued by the same issue.